src/XIVLauncher.Core/Components/MainPage/MainPage.cs

```diff
@@ -692,6 +692,9 @@ public async Task<Process> StartGameAndAddon(Launcher.LoginResult loginResult, b
             var wineLogFile = new FileInfo(Path.Combine(App.Storage.GetFolder("logs").FullName, "wine.log"));
             runner = new UnixGameRunner(App.Settings.WineStartupType ?? WineStartupType.Command, App.Settings.WineStartCommandLine, Program.CompatibilityTools, App.Settings.DxvkHudType,
                 App.Settings.WineDebugVars ?? string.Empty, wineLogFile, dalamudLauncher, dalamudOk);
+
+            App.Settings.AdditionalArgs += $" UserPath={Program.CompatibilityTools.UnixToWinePath(App.Settings.GameConfigPath.FullName)}";
```

Comment:
Did you test this? If you do it like that, every time the game is started, the UserPath argument will be added to the setting and saved to the config. Check the code sample from my last review.

Reply:
I did test this, and it does work, but I didn't realize this was being written to the .ini file. Your changes are the same as mine though so it looks good to me.